samples/fpga/gemm_systolic_vectorized.py

```diff
@@ -403,8 +403,8 @@ def make_sdfg(specialized, vec_width):
         sdfg = make_sdfg(True, vec_width)
         sdfg.specialize(dict(P=P, M=M, N=N, K=K))
 
-    print("Matrix multiplication {}x{}x{} with {} PEs ({}specialized)".format(
-        M.get(), N.get(), K.get(), P.get(),
+    print("Matrix multiplication {}x{}x{} with {} PEs and vectorization width {} ({}specialized)".format(
+        N.get(), K.get(), M.get(), P.get(), args["W"],
```

Comment:
The comment is about the correctness check done after the program execution, which considers `N` and `K` as the size of the result matrix. They should be `N` and `M`

(I cannot add the comment on the right line)